cli/util/uploader.go

```diff
@@ -109,6 +103,27 @@ func uploadBin(ctx *cli.Context) error {
 		return cli.Exit(fmt.Errorf("failed to get network info: %w", err), 1)
 	}
 	homomorphicHashingDisabled := net.HomomorphicHashingDisabled()
+	container, err := p.ContainerGet(ctx.Context, containerID, client.PrmContainerGet{})
```

Comment:
I'm starting to be extremely suspicious regarding all operations with pool. Can we use retry here?

Reply:
also we can add retry here: `net, err := p.NetworkInfo(ctx.Context, client.PrmNetworkInfo{})` will do it in https://github.com/nspcc-dev/neo-go/pull/3638#discussion_r1812175065